src/app/modules/onboarding/module.nim

```diff
@@ -76,7 +76,11 @@ method delete*[T](self: Module[T]) =
   self.controller.delete
 
 method onAppLoaded*[T](self: Module[T], keyUid: string) =
```

Comment:
What about the `keyUid` param, still needed?

Reply:
I guess since I don't use it, we can remove it. It was actually never used in the current onboarding. I assume it's a leftover from the previous onboarding

Reply:
Yeah, it indeed seems unused; at one point we thought it'd be useful as a moment to save the credentials into the keychain right @igor-sirotin ?